expo/expo
# [web][image-picker] Fix `base64` return

# Why
`ImagePicker` doesn't return the `base64` of the image as stated in the [documentation](https://docs.expo.io/versions/latest/sdk/imagepicker/#returns-4).

# How
The base64 of the image is returned if the `base64` param is passed as `true` to `readFile()` (which is called by `openFileBrowserAsync()`).

# Test Plan
- Navigate to `cd apps/bare-expo`
- Run `yarn web`
- Press on the _APIs_ tab
- Press on _ImagePicker_
- Toggle the _With base64_ switch
- Press on _Pick photo_ or any other option that opens the file browser for image selection
- Select an image, then confirm
- You should see the base64 for the selected image in the object under the image and in the console inside _Results_ 

### Before
<img width="1792" alt="Screenshot 2021-04-14 at 09 32 18" src="https://user-images.githubusercontent.com/25615674/114671784-c118f380-9d04-11eb-8041-af71fcba6574.png">

### After
<img width="1792" alt="Screenshot 2021-04-14 at 09 30 06" src="https://user-images.githubusercontent.com/25615674/114671832-cece7900-9d04-11eb-8e91-e6656563aafa.png">
